src/Components/Blazor/testassets/Wasm.Authentication.Client/Pages/Counter.razor

```diff
@@ -0,0 +1,16 @@
+﻿@page "/counter"
```

Comment:
Seems unlikely that this is necessary for this particular testassets project. Could we avoid duplicating unnecessary parts of the project template? The reason is:

 * It bloats the repo, making it harder to find things you're looking for due to getting lots of duplicate matches
 * Each time we change the template, people have to make choices about which of the test assets should similarly be updated, and if they don't update the copies then they start to see neglected and out-of-date.

There's no need, for example, for this project to contain any CSS styling or use of layouts.

Reply:
removed